store_app/views/login_session.py

```diff
@@ -0,0 +1,50 @@
+from flask import make_response, jsonify, request, session, url_for, redirect, g
+from tables.models import User
+from app import app
+
+
+@app.before_request
+def before_request():
+    g.user = None
+    if 'user_id' in session:
+        user = User.query.get(session['user_id'])
+        g.user = user
+
+
+@app.route('/login', methods=['POST'])
+def login():
+    try:
+        session.pop('user_id', None)
```

Comment:
Just to make sure that you are aware of it, `session` from Flask relies on the flask app `secret` and my guess is that if the secret is disclosed, it would be possible to hijack this mechanism, with obvious security risks.

I really like that you added that after the latest feedback though, it is more to tell you that this is not production ready 👍

Reply:
I had added the app secret key in app.py. I thought only adding that was required. Do I have use that key in other places as well or only defining it was enough?